desktop-exporter/server.go

```diff
@@ -18,13 +18,13 @@ func getTracesHandler(traceStore *TraceStore) func(http.ResponseWriter, *http.Re
 		traceStore.mut.Lock()
 		defer traceStore.mut.Unlock()
 
-		writer.WriteHeader(http.StatusOK)
-
-		traces, err := json.Marshal(traceStore.traceMap)
+		jsonTraces, err := json.Marshal(traceStore.traceMap)
 		if err != nil {
-			fmt.Printf("error marshalling traceStore: %s\n", err)
+			panic(fmt.Errorf("error marshalling traceStore: %s\n", err))
 		} else {
-			fmt.Fprintf(writer, "Hello traceStore:\n%s\n", traces)
+			writer.WriteHeader(http.StatusCreated)
```

Comment:
```suggestion
			writer.WriteHeader(http.StatusOK)
```

`http.StatusCreated` is usually used as a response code to a `POST` request creating / storing some bit of data. Here we are fetching data with a `GET` request, so a `200` or `http.StatusOK` is what we want.

https://go.dev/src/net/http/status.go

Reply:
Ok!